release/release.go

```diff
@@ -90,6 +91,12 @@ type PlatformGcp struct {
 	Image *GcpImage `json:"image"`
 }
 
+// PlatformIBMCloud IBMCloud/PowerVS image detail
+type PlatformIBMCloud struct {
+	PlatformBase
+	Images map[string]IBMCloudObject `json:"images"`
```

Comment:
using Images here as discussed in: https://github.com/coreos/coreos-assembler/pull/2361#discussion_r713377596